### PR TITLE
[MetroHash] Add new port

### PR DESCRIPTION
**Pull request by kreuzerkrieg**
- What does your PR fix? 
MetroHash port
- Which triplets are supported/not supported?
Any except UWP
- Have you updated the CI baseline?
Yes
- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

**Comment by JackBoosY** [comment]
I found that `MetroHash` failed to build on `x86-windows`, `x64-windows` and `arm64-windows`, please get the logs [here](https://dev.azure.com/vcpkg/public/_build/results?buildId=28934&view=artifacts&type=publishedArtifacts).

Thanks.

**Comment by kreuzerkrieg** [comment]
x64-windows fixed
the rest is disabled

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
LGTM, thanks for this PR!

**Comment by kreuzerkrieg** [comment]
Anything prevents this PR from being merged?

**Comment by JackBoosY** [comment]
@strega-nil Ready to merge.

File: ports/metrohash/CONTROL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Source: metrohash
+Version: 1.1.3
+Homepage: https://github.com/jandrewrogers/MetroHash
+Description: MetroHash is a set of state-of-the-art hash functions for non-cryptographic use cases.
```
Comment (strega-nil) [inline ports/metrohash/CONTROL]: Could you add a `Supports` clause here?

**Review by strega-nil — CHANGES_REQUESTED** [review]
Looks good to me! Just add the `Supports` clause and I'll merge :D

File: ports/metrohash/CONTROL (hunk 1 above)
Comment (kreuzerkrieg) [reply ports/metrohash/CONTROL]: Done

**Comment by strega-nil** [comment]
Thanks much!

**Pull request merged**